acisfp_check/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "2.2.0"
+__version__ = "2.2.1"
```

Comment:
Being pedantic, this PR changes the API so should go to version "2.3".  (By making `search_obsids_for_viols` into a method and changing the call signature).